GrassChoppers/Modules/Home/HomeViewController.swift

```diff
@@ -47,5 +51,9 @@ class HomeViewController: UIViewController {
 }
 
 extension HomeViewController: HomeLogicControllingDelegate {
-    
+    func update(name: String) {
+        
+        label.text = name
```

Comment:
Here, `HomeViewController` is conforming to the protocol `HomeLogicControllingDelegate` so that the view controller can act as the logic controller's delegate. This will allow us to receive updates from the logic controller when there is new data.

Since `HomeLogicControllingDelegate` declares that we need to define `func update(name: String)` (see [here](https://github.com/jwitcig/GrassChoppers/pull/2/files#diff-95819295f08c522fd5948df08c71a1bbR9)), we have to implement it on `HomeViewController`, or else Swift will complain.

Once we implement `func update(name: String)`, we're ready to receive updates from LogicController.
When LogicController gets new data from DataManager,[ it forwards it to ViewController](https://github.com/jwitcig/GrassChoppers/pull/2/files#diff-95819295f08c522fd5948df08c71a1bbR44) (via its delegate...which _is_ ViewController), and that message is received by ViewController here. Lastly, we update the label with the new data.